File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/model/Address.kt
package com.maxholmes.androidapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Address(
    var id: String,
    var city: String,
    var district: String,
    var ward: String,
    var street: String
) : Parcelable

File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/Entities/Address.cs
namespace PickleballCourtBookingSystem.Core.Entities

{
    public class Address
    {
        public Guid? Id { get; set; }
        public string? City { get; set; }
        public string? District { get; set; }
        public string? Ward { get; set; }
        public string? Street { get; set; }
    }
}
